vendors/swedbank-pay-sdk-php/tests/CardPaymentTest.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+use SwedbankPay\Api\Service\Creditcard\Request\Purchase;
+use SwedbankPay\Api\Service\Creditcard\Request\Verify;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentPurchaseCreditcard;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentPurchaseObject;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentUrl;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentVerifyCreditcard;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentVerifyObject;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentPurchase;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentVerify;
+use SwedbankPay\Api\Service\Payment\Resource\Collection\PricesCollection;
+use SwedbankPay\Api\Service\Payment\Resource\Collection\Item\PriceItem;
+use SwedbankPay\Api\Service\Creditcard\Resource\Request\PaymentPayeeInfo;
+
+use SwedbankPay\Api\Service\Data\ResponseInterface as ResponseServiceInterface;
+use SwedbankPay\Api\Service\Resource\Data\ResponseInterface as ResponseResourceInterface;
+
+class CardPaymentTest extends TestCase
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class CardPaymentTest has a coupling between objects value of 15. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)